packages/react/tabs/src/Tabs.tsx

```diff
@@ -157,7 +156,6 @@ const TabsTrigger = React.forwardRef<TabsTriggerElement, TabsTriggerProps>(
     const triggerId = makeTriggerId(context.baseId, value);
     const contentId = makeContentId(context.baseId, value);
     const isSelected = value === context.value;
-    const handleTabChange = useCallbackRef(() => context.onValueChange(value));
```

Comment:
Remove the `useCallbackRef` dep while I was here as I couldn't work out why it was necessary?

Reply:
Good spot, looks unnecessary to me as well 🤔